doc/source/networks.rst

```diff
@@ -11,15 +10,54 @@ Manager methods
 Networks can be queried through the following client manager
 methods:
 
```

Comment:
Probably need a note here to say that the 'writable/savable' attributes are only available with the `network` api extension, and this is indicated by the `has_network_api_extension()`  (or whatever the function may get called!).

Reply:
Of course, I wasn't even aware about this function - I'll add these informations